bodhi/server/templates/update.html

```diff
@@ -692,6 +692,23 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              % if request.registry.settings.get('ci.required', 'False') == 'True':
```

Comment:
Following #1489 , this got simplified to ``% if request.registry.settings.get('ci.required'):``